reference-implementation/lib/abstract-ops/readable-streams.js

```diff
@@ -1584,6 +1584,8 @@ function ReadableByteStreamControllerPullInto(controller, view, min, readIntoReq
   assert(minimumFill >= elementSize && minimumFill <= view.byteLength);
   assert(minimumFill % elementSize === 0);
 
+  const byteOffset = view.byteOffset;
+  const byteLength = view.byteLength;
```

Comment:
The reference implementation was doing [steps 8 and 9](https://streams.spec.whatwg.org/commit-snapshots/4f2a2c1012342885b2c2d7608fa58ef81631b058/#readable-byte-stream-controller-pull-into) *after* calling `TransferArrayBuffer`, which meant that `view.byteLength` was already `0`. We didn't notice this, because we weren't *actually* transferring before. 😛

Fortunately, all "real" implementations already do this correctly.